doc/architecture/decisions/0003-store-host-output-file-in-etl-container.md

```diff
@@ -0,0 +1,27 @@
+# 3. Store Host output file in ETL container
+
+Date: 2017-06-07
+
+## Status
+
+Accepted
+
+## Context
+
+The ETL process takes a long time to run, approximately 6 hours.
+Running the ETL process makes close to 30,000 requests to the Syndication API.
+Several applications need access to the JSON file created by the ETL.
+
+## Decision
+
+Rather than each application running its own copy of the ETL to obtain the Syndication data in JSON format, a single instance of the ETL
+will run and provide access to the resultant file via an nginx web server running in the ETL container.
+
+The output JSON is hosted in the container to reduce complexity and reliance on external solutions, such as Azure.
```

Comment:
I'm not sure this approach reduces complexity, I thought it was done solely based on removing the reliance of third party systems? Perhaps remove that bit?